test/NuGet.Core.Tests/NuGet.Common.Test/RuntimeEnvironmentHelperTests.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace NuGet.Common.Test
+{
+    public class RuntimeEnvironmentHelperTests
+    {
+        [Fact]
+        public void RuntimeEnvironmentHelper_MutuallyExclusiveRuntimes()
```

Comment:
The values are not mutually exclusive.  On Windows, `IsWindows` returns `true`, but `IsMacOSX` and `IsLinux` both return `false`.  `IsMacOSX` and `IsLinux` values are not mutually exclusive.

A better name for this test would be:

    RuntimeEnvironmentHelper_OperatingSystemChecks_OnlyOneReturnsTrue